blueman/main/Tray.py

```diff
@@ -0,0 +1,44 @@
+from importlib import import_module
+
+import logging
+
+from blueman.Functions import check_single_instance
+from blueman.main.AppletService import AppletService
+
+import gi
+gi.require_version('Gtk', '3.0')
+from gi.repository import Gtk
+
+
+class BluemanTray(object):
+    def __init__(self):
+        check_single_instance("blueman-tray")
+
+        logging.basicConfig(level=logging.INFO)
```

Comment:
This should eventually (when logging stuff is merged) be moved to app/blueman-tray and add the arguments needed for setting log level.